.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 READ ME FIRST: Please answer *both* questions below and check off every point from the essential checklist!
 -->
 
-1. This PR fixes #[fill_in_number_here].
+1. This PR fixes or fixes partly #[fill_in_number_here].
```

Comment:
Could you make it "fixes part of"? (That's more grammatical.)

Reply:
Also -- just to check: for PRs that actually fix the issue, don't we want to see "fixes #XXX" in the PR, so that the original issue does get closed? Given the current wording, that won't happen, if I'm reading correctly.

Reply:
We might have a chance if the PR title is formatted properly. Github used to autoclose issues if the keywords where in the title or description. I couldn't find documentation on that behavior anymore though...

Reply:
Also changed the wording above.